### ISSUE TITLE
if `rootDir` is not specified parent folder is not determined properly

### ISSUE THREAD

**Issue by beltekylevi** [MelvinVermeer/eslint-plugin-no-relative-import-paths#9]
If `rootDir` is not specified the rule does not work properly. To be precise `isParentFolder` doesn't work properly if I'm not mistaken.

```js
import something from '../something';
```

This should be caught but isn't. 
[I've set up an example repository.](https://github.com/beltekylevi/eslint-plugin-no-relative-import-paths-issue)

**Comment by beltekylevi** [comment]
If `rootDir` is not specified the rule does not work properly. To be precise `isParentFolder` doesn't work properly if I'm not mistaken.

```js
function isParentFolder(relativeFilePath, context, rootDir) {
  const absoluteRootPath = context.getCwd() + (rootDir !== '' ? path.sep + rootDir : '');
  const absoluteFilePath = path.join(path.dirname(context.getFilename()), relativeFilePath)

  return relativeFilePath.startsWith("../") && (
    rootDir === '' || // < --- because of this
    (absoluteFilePath.startsWith(absoluteRootPath) &&
    context.getFilename().startsWith(absoluteRootPath))
  );
}
```

**Comment by beltekylevi** [comment]
I've updated my issue accordingly. I'm happy to open a pull request later if you have no time to fix this. :)

**Comment by MelvinVermeer** [comment]
> I've updated my issue accordingly. I'm happy to open a pull request later if you have no time to fix this. :)

@beltekylevi, feel free to submit a pr for this.

**Comment by awesometic** [comment]
Oh, I don't know that this issue is already reported. 😮 

I just tried this package today and I found the same issue in my project, so I have fixed and made a patch for fixing this one.

Here is my PR. Will update its description.
https://github.com/MelvinVermeer/eslint-plugin-no-relative-import-paths/pull/10

I hope I didn't interfere with you making a patch... 😅

**Comment by beltekylevi** [comment]
> Oh, I don't know that this issue is already reported. 😮
> 
> I just tried this package today and I found the same issue in my project, so I have fixed and made a patch for fixing this one.
> 
> Here is my PR. Will update its description. #10
> 
> I hope I didn't interfere with you making a patch... 😅

It was in my backlog to fix this issue so no worries you did not interfere. 😀 Thank you for fixing it!

**Comment by MelvinVermeer** [comment]
PR is merged, i will release it immediately! Nice work
